markdown/setup/dbvisualizer-driver-setup.md

```diff
@@ -0,0 +1,42 @@
+### DbVisualizer 
+[Link to product webpage](https://www.dbvis.com/download/).
+
+#### Adding the Amazon Timestream JDBC Driver
+1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`)
+2. Start the DbVisualizer application and navigate to the menu path: **Tools > Driver Manager...**. In the popped up window, click the "Custom" link on the right side.
+
+    ![Example](../images/dbvisualizer/DbVisualizer1.png)
```

Comment:
It would be good to highlight on the screenshots what the user should click on, so instructions are easy to use.

Reply:
Done